### ISSUE TITLE
ImpulseTracker support

### ISSUE THREAD

**Issue by jarotg** [jhalme/webaudio-mod-player#18]
Are there plans to support this format (.it)?

**Comment by jhalme** [comment]
Yes - I've already done a few bits of code for .it support. However, I'm super busy with other stuff at the moment, so it's on hold for the time being.

**Comment by esaruoho** [comment]
Hi, I'd be terribly interested in seeing .IT support, is there anything that can be done to help it? @jhalme
